### ISSUE TITLE
Add functionality to quickly locate picker name by Order ID

### ISSUE THREAD

**Issue by Adameddy26** [hotwax/fulfillment#837]
## Description
Currently, when a user clicks on a picker name chip, all orders assigned to that picker are listed. However, there is no direct way to locate the picker responsible for a specific order using the Order ID.
For example, if the Order ID is UAT12538, the user has to manually click on each picker name chip and search for the desired order in the filtered list. This process can be time-consuming, especially when managing a large number of orders.

## Proposed Solution:
- [ ] Order Search with Picker Information:
Add a search field where the user can enter an Order ID (e.g., UAT12538).
The system should return the picker name directly associated with that Order ID, allowing the user to avoid clicking each picker name chip manually.
 - [ ] Picker Name Display in Order Details:
When viewing detailed information about an order, the picker name should be prominently displayed, so the user can immediately identify who is responsible for the order.

**Comment by dt2patel** [comment]
If the picker needs to be inspected, click on the order detail page. The current chip at the top of the order should be altered in the following way:

Currently, clicking on it, it opens the pick list.
<img width="839" alt="Image" src="https://github.com/user-attachments/assets/c6222275-4147-476c-a7f2-7f97b8782e56" />

New:
Show the first picker name and then +<count> for additional pickers. Clicking on it will give the following popup

<img width="429" alt="Image" src="https://github.com/user-attachments/assets/4632704a-dbee-45b7-8dd9-5975b1457279" />

**Comment by ravilodhi** [comment]
> If the picker needs to be inspected, click on the order detail page. The current chip at the top of the order should be altered in the following way:
> 
> Currently, clicking on it, it opens the pick list. <img alt="Image" width="839" src="https://private-user-images.githubusercontent.com/60128132/428232299-c6222275-4147-476c-a7f2-7f97b8782e56.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY3MDk2NDUsIm5iZiI6MTc1NjcwOTM0NSwicGF0aCI6Ii82MDEyODEzMi80MjgyMzIyOTktYzYyMjIyNzUtNDE0Ny00NzZjLWE3ZjItN2Y5N2I4NzgyZTU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTAxVDA2NDkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyY2JlZjViYmMyZGUzODBiMzgwMzQxMzVkNjg4MzhkNmU3MDFjMjA3MGU1ZjAzZjhhZTE0MGZjOThkZDEzMzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.p4TX_u8Ug1-XlwjBkn_QA_N7su8rZcYxQzNeDJq8UAQ">
> 
> New: Show the first picker name and then + for additional pickers. Clicking on it will give the following popup
> 
> <img alt="Image" width="429" src="https://private-user-images.githubusercontent.com/60128132/428236740-4632704a-dbee-45b7-8dd9-5975b1457279.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY3MDk2NDUsIm5iZiI6MTc1NjcwOTM0NSwicGF0aCI6Ii82MDEyODEzMi80MjgyMzY3NDAtNDYzMjcwNGEtZGJlZS00NWI3LThkZDktNTk3NWIxNDU3Mjc5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTAxVDA2NDkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRhZjU0NzZmNTUwNjlmZjFkNDAzZjljN2ZiZDY2YmY5MGU3OTA5ZWZiYzAzOTRjYWVjYjI2Yjg2NWQ1M2VkMmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.sFFNmVI-WqhhlDcbmaheKdZOPR9pH20nqK6DORdfBuw">

As we have already removed the Picklist QR Code functionality from the In Progress Orders page, the QR Code action option will also be removed from this popup, and only the View Picklist option will remain. Should it directly open the picklist PDF?
@dt2patel

**Comment by dt2patel** [comment]
Yes make sense. we can close this ticket until then I guess? If so, @ravilodhi please close it

**Comment by ravilodhi** [comment]
As QR code functionality is currently removed and the picker name is not returned in the order detail API, which will be included in the new API we are integrating into the fulfillment order detail page soon, we are closing this ticket until then.
